ukrainian/wait-more/out/try/index.md

```diff
@@ -1,2 +1,5 @@
 Ви берете кілька листків с кущика, тягнете до рота та відчуваєте як ваше обличчя перекошує. 
-Яке ж воно кисле! Цікаво, воно все кисле, чи різні частини куща мають різні смаки.. Спробуємо?
\ No newline at end of file
+Яке ж воно кисле! Цікаво, воно все кисле, чи різні частини куща мають різні смаки.. Спробуємо?
+[Спробувати ще один листочок](parts/leaf/leaf.md)
+[Спробувати корінь](parts/root/root.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.